blessings/__init__.py

```diff
@@ -420,7 +422,7 @@ def derivative_colors(colors):
                       'shadow', 'standout', 'subscript', 'superscript']))
 
 
-class ParametrizingString(unicode):
+class ParametrizingString(six.text_type):
```

Comment:
My only comment at a glance is to import these symbols more deeply to match the rest of them, as in `from six import text_type, ...`.

Reply:
I'm entirely not sure why, but doing this fails for me when running the tests, with the following traceback:

```
GLOB sdist-make: /Users/dhunt/workspace/blessings/setup.py
py27 inst-nodeps: /Users/dhunt/workspace/blessings/.tox/dist/blessings-1.6.1.zip
py27 installed: blessings==1.6.1,nose==1.3.7,six==1.11.0
py27 runtests: PYTHONHASHSEED='3106176075'
py27 runtests: commands[0] | nosetests blessings
...............S...E...
======================================================================
ERROR: Make sure you get nice hints if you misspell a formatting wrapper.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/dhunt/workspace/blessings/.tox/py27/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/Users/dhunt/workspace/blessings/blessings/tests.py", line 232, in test_nice_formatting_errors
    t.bold_misspelled('hey')
  File "/Users/dhunt/workspace/blessings/blessings/__init__.py", line 466, in __call__
    if len(args) == 1 and isinstance(args[0], string_types):
NameError: global name 'string_types' is not defined

----------------------------------------------------------------------
Ran 23 tests in 0.006s

FAILED (SKIP=1, errors=1)
ERROR: InvocationError for command '/Users/dhunt/workspace/blessings/.tox/py27/bin/nosetests blessings' (exited with code 1)
```

It's the same for all environments.

Reply:
Hmm, I tried it locally, and it worked for me. Maybe if you push it, it'll work on Travis.